opengrok-indexer/src/main/java/org/opengrok/indexer/index/PendingFileCompleter.java

```diff
@@ -372,11 +373,16 @@ private void doLink(PendingSymlinkageExec lnk) throws IOException {
                     sourceParentFile.exists()) {
                 Files.createSymbolicLink(sourcePath, Paths.get(lnk.targetRel));
             }
+        } catch (FileAlreadyExistsException e) {
```

Comment:
is the double check for `exists` neccessary now?

Reply:
yes, that's for the parent directory :-)

Reply:
aha, ok.